src/main/java/core/basesyntax/model/Parser.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax.model;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Parser {
+    private static final String REGEX = ",";
+    private static final int OPERATION = 0;
+    private static final int FRUIT = 1;
+    private static final int QUANTITY = 2;
+
+    public static List<FruitTransaction> parseListToTransactionList(List<String> list) {
+        List<FruitTransaction> fruitTransactionList = new ArrayList<>();
+        for (int i = 1; i < list.size(); i++) {
```

Comment:
```suggestion
        for (String line : list) {
```

Reply:
Isn't here possible for use fori? Cause when I replaced it with foreach, I forced to do hardcode...